virttest/libvirt_version.py

```diff
@@ -47,6 +54,9 @@ def version_compare(major, minor, update):
 
     compare_version = major * 1000000 + minor * 1000 + update
 
-    if LIBVIRT_LIB_VERSION >= compare_version:
+    LIBVIRT_LIB_VERSION_tmp = LIBVIRT_LIB_VERSION
+    if session:
+        LIBVIRT_LIB_VERSION = LIBVIRT_LIB_VERSION_ORG
+    if LIBVIRT_LIB_VERSION_tmp >= compare_version:
```

Comment:
I do not think 'LIBVIRT_LIB_VERSION_tmp' is needed.
```
if LIBVIRT_LIB_VERSION >= compare_version:
    if session:
        LIBVIRT_LIB_VERSION = LIBVIRT_LIB_VERSION_ORG
    return True
```

Reply:
@dzhengfy Thanks! updated.

Reply:
@dzhengfy Rechecked the code,  LIBVIRT_LIB_VERSION should be restored in all cases but not  under 'if LIBVIRT_LIB_VERSION >= compare_version' section.  I have removed 'backup' and 'restore' actions as chunfu's comments.